specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/azureFirewall.json

```diff
@@ -473,6 +473,128 @@
          },
          "description":"Properties of an application rule."
       },
+      "AzureFirewallNatRuleCollectionPropertiesFormat": {
```

Comment:
nit: Please rename this to `AzureFirewallNatRuleCollectionProperties` for brevity